pages/linux/trace-cmd.md

```diff
@@ -14,7 +15,7 @@
 
 - Start tracing with a specific plugin:
 
-`trace-cmd start -p {{timerlat|osnoise|hwlat|blk|mmiotrace|function_graph|wakeup_dl|wakeup_rt|wakeup|function|nop}}`
+`trace-cmd start -p {{plugin}}`
```

Comment:
This is probably better if you just pruned the list to the most important onesand made it `{{plugin1|pluging2|plugin3|...}}`